src/main/java/io/wisetime/connector/jira/JiraConnector.java

```diff
@@ -77,7 +77,8 @@ public void performTagUpdate() {
 
       final List<Issue> issues = jiraDao.findIssuesOrderedById(
           lastPreviouslySyncedIssueId.orElse(0L),
-          tagUpsertBatchSize()
+          tagUpsertBatchSize(),
+          onlyUpsertTagsForProjectKeys()
```

Comment:
we always call `onlyUpsertTagsForProjectKeys` whenever connector will retrieve jira issues but the method will always return the same data. Maybe we can just initialize projects keys once?

Reply:
Have added memoization to method that loads the config.

Reply:
Have reverted memoization - makes testing convoluted. Let's keep things simple.